extensions/vscode/src/templates/bloc-event.template.ts

```diff
@@ -29,3 +35,15 @@ function getDefaultBlocEventTemplate(blocName: string): string {
 abstract class ${pascalCaseBlocName}Event {}
 `;
 }
+
+function getFreezedBlocEvent(blocName: string): string {
+  const pascalCaseBlocName =
+    changeCase.pascalCase(blocName.toLowerCase()) + "Event";
+  const snakeCaseBlocName = changeCase.snakeCase(blocName.toLowerCase());
+  return `part of '${snakeCaseBlocName}_bloc.dart';
+
+@freezed
+abstract class ${pascalCaseBlocName} with _\$${pascalCaseBlocName} {
+  const factory ${pascalCaseBlocName}.event1() = Event1;
```

Comment:
```suggestion
  const factory ${pascalCaseBlocName}.started() = _Started;
```
To follow the example https://bloclibrary.dev/#/blocnamingconventions?id=anatomy